Description: fix build with kernel >= 3.2.x
 but don't break build with lernel < 3.2
Bug: http://code.google.com/p/r8168/issues/detail?id=8
Last-Updated: 2012-01-28

Index: b/src/r8168_n.c
===================================================================
--- a/src/r8168_n.c	2011-12-14 13:39:56.378775542 +0100
+++ b/src/r8168_n.c	2012-01-28 02:18:56.370222297 +0100
@@ -14282,7 +14282,11 @@
 
 		txd = tp->TxDescArray + entry;
 		len = frag->size;
+#if LINUX_VERSION_CODE < KERNEL_VERSION(3,2,0)
 		addr = ((void *) page_address(frag->page)) + frag->page_offset;
+#else
+		addr = ((void *) page_address(frag->page.p)) + frag->page_offset;
+#endif
 		mapping = pci_map_single(tp->pci_dev, addr, len, PCI_DMA_TODEVICE);
 
 		/* anti gcc 2.95.3 bugware (sic) */
